dali/pipeline/operators/expressions/arithmetic_meta.h

```diff
@@ -336,6 +347,25 @@ inline ArithmeticOp NameToOp(const std::string &op_name) {
   return it->second;
 }
 
+inline DALIDataType TypeNameToTypeId(const std::string &type_name) {
+  static std::map<std::string, DALIDataType> token_to_type_id = {
+    std::make_pair("uint8", DALIDataType::DALI_UINT8),
+    std::make_pair("uint16", DALIDataType::DALI_UINT16),
+    std::make_pair("uint32", DALIDataType::DALI_UINT32),
+    std::make_pair("uint64", DALIDataType::DALI_UINT64),
+    std::make_pair("int8", DALIDataType::DALI_INT8),
+    std::make_pair("int16", DALIDataType::DALI_INT16),
+    std::make_pair("int32", DALIDataType::DALI_INT32),
+    std::make_pair("int64", DALIDataType::DALI_INT64),
+    std::make_pair("float16", DALIDataType::DALI_FLOAT16),
+    std::make_pair("float32", DALIDataType::DALI_FLOAT),
+    std::make_pair("float64", DALIDataType::DALI_FLOAT64)
```

Comment:
Is `std::make_pair` really necessary? Don't brace-enclosed initializers work here?

Reply:
My bad, it works. Done.